src/app/public/modules/auth-http-client/auth-interceptor.spec.ts

```diff
@@ -239,6 +248,51 @@ describe('Auth interceptor', () => {
     interceptor.intercept(request, next).subscribe(() => {});
   });
 
+  it('should convert tokenized urls and honor the hard-coded zone, even with a port added', (done) => {
+    const interceptor = createInteceptor();
+
+    const request = createRequest(
+      true,
+      '1bb://eng-hub00-pusa01:8080/version'
+    );
+
+    validateAuthRequest(done, (authRequest) => {
+      expect(authRequest.url).toBe('https://eng-pusa01.app.blackbaud.net/hub00/version');
+    });
+
+    interceptor.intercept(request, next).subscribe(() => {});
+  });
+
+  it('should convert tokenized urls when serving locally without a port provided', (done) => {
```

Comment:
@Blackbaud-ColbyWhite this is the scenario you thought was missing in the other spec file

Reply:
I want to leave this one here because when the service discovery happens/some other kind of augmentation happens I'd want the lack of port but being served locally to invoke the proper fallback function as configured by this spec.

Reply:
👍